### PR TITLE
fix: update size depending on wrapping component

### LINKED ISSUE #3: Update size depending on component size

### PR DESCRIPTION
**Pull request by pyphilia**
close #3

### CODE REVIEW

File: src/components/modes/teacher/widgets/util/common.js [hunk 1]
```diff
@@ -210,7 +213,13 @@ export const combineContents = (listOfContent) => {
 }
 
 export const formatChunkDate = (group) =>
-  group.length > 1 ? `${group[0]} - ${group[group.length - 1]}` : group[0]
+  group.length > 1
+    ? `${group[0]}${DATE_SEPARATOR}${group[group.length - 1]}`
+    : group[0]
+
+export const dateAreChunked = (dates) => {
```
Comment (juancarlosfarah) [inline src/components/modes/teacher/widgets/util/common.js]: Change to `datesAreChunked`.

File: .eslintrc [hunk 2]
```diff
@@ -20,7 +20,7 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "import/no-unresolved":false,
+    "import/no-unresolved":"off",
```
Comment (juancarlosfarah) [inline .eslintrc:23]: Why do we need this off?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks for this PR. Just a minor typo and a question.

File: .eslintrc (hunk 2 above)
Comment (pyphilia) [reply .eslintrc:23]: It is the new syntax, you can choose either `off` or `0` https://eslint.org/docs/user-guide/configuring#configuring-rules

**Pull request merged**